master/docs/relnotes/0.9.0b7.rst

```diff
@@ -0,0 +1,75 @@
+Release Notes for Buildbot 0.9.0b7
+==================================
+
+
+The following are the release notes for Buildbot 0.9.0b7
+This version was released on February 14, 2016.
+
+See :ref:`Upgrading to Nine` for a guide to upgrading from 0.8.x to 0.9.x
+
+Master
+------
+
+Features
+~~~~~~~~
+
+Fixes
+~~~~~
+
+* Fix incompatibility with MySQL-5.7 (:bug:`3421`)
+* Fix incompatibility with postgresql driver psycopg2 (:bug:`3419`, further regressions will be catched by travis)
+* Fix regressions in forcescheduler UI (:bug:`3416`, :bug:`3418`, :bug:`3422`)
+
+Deprecations, Removals, and Non-Compatible Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* The ``buildbot`` Python dist now (finally) requires SQLAlchemy-0.8.0 or later and SQLAlchemy-Migrate-0.9.0 or later.
+  While the old pinned versions (0.7.10 and 0.7.2, respectively) still work, this compatibility is no longer tested and this configuration should be considered deprecated.
```

Comment:
The second sentence here isn't actually true -- we have circular dependency errors with that version combination.

Should we just swallow the pill and force our users to upgrade?